elections/steering/2022/README.md

```diff
@@ -0,0 +1,308 @@
+# 2021 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
```

Comment:
Can we call out that this is stubbed/draft? Or maybe just drop this file completely if its not up to date?